### ISSUE TITLE
This My problem after update 

### ISSUE THREAD

**Issue by johnvfitness** [InstaPy/InstaPy#2058]
Traceback (most recent call last):
  File "john.py", line 45, in <module>
    session.follow_user_followers(['tiboinshape', 'tavicastro','jujufitcats','justinegallice','stanimal9','antonao','carolineclnfit', 'fitbyclem','chloe_fit_','mmjhottie'], amount=10, randomize=True, interact=True, sleep_delay=10)
  File "/Users/jonathanverdier/Desktop/InstaPy/instapy/instapy.py", line 1864, in follow_user_followers
    self.logfolder)
  File "/Users/jonathanverdier/Desktop/InstaPy/instapy/unfollow_util.py", line 750, in get_given_user_followers
    logger, logfolder)
  File "/Users/jonathanverdier/Desktop/InstaPy/instapy/unfollow_util.py", line 575, in get_users_through_dialog
    quick_button = follow_buttons[quick_index]
IndexError: list index out of range

**Comment by johnvfitness** [comment]
And
Traceback (most recent call last):
  File "john.py", line 45, in <module>
    session.like_by_locations(['213633143'], amount=10)
  File "/Users/jonathanverdier/Desktop/InstaPy/instapy/instapy.py", line 797, in like_by_locations
    self.logger)
TypeError: validate_username() missing 1 required positional argument: 'logger'
Jonathans-MacBook-Pro:instapy jonathanverdier$

**Comment by uluQulu** [comment]
Hi @johnvfitness,
Thanks for catching it!
```erlang
quick_button = follow_buttons[quick_index]
IndexError: list index out of range
```
Actually, I've thought `random.randint(z, x)` generates a value in [z, x) range but it is [z, x], which is what misbehaved. So I have added it's fix in #2042 05998f9807699ad508508f09174ecf8d5916a37b.

And the other issue,
```erlang
TypeError: validate_username() missing 1 required positional argument: 'logger'
```
is already fixed in #2042, see https://github.com/timgrossmann/InstaPy/pull/2042#issuecomment-390385171 to apply that particular fix manually.

Cheers!
